app/src/debug/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">DEBUG.MaterialAndroidBootstrap</string>
```

Comment:
Maybe, but as you also change app logo, we can easly see which app is debug and which is release